### DISCUSSION TITLE
PayPal e-mail to donate to?

### DISCUSSION THREAD

**Discussion by 38github** [johang/sd-card-images#127] [General]
I would like to donate a few kronor for the great job you do. What e-mail can I do this to? Regards.

**Comment by johang** [comment]
It's not hard to find my email address, but I prefer that you donate to some charity instead.

**Comment by 38github** [comment]
Any particular charity you would like to see receive donations?

21 apr. 2024 16:00 av ***@***.***:
>
> It's not hard to find my email address, but I prefer that you donate to some charity instead.
>
>
